### DISCUSSION TITLE
Border gradient animation

### DISCUSSION THREAD

**Discussion by beckend** [YaLTeR/niri#725] [Ideas]
In hyprland you can configure:

```
  animation = borderangle, 1, 15, linear, loop
```

Description from their README
borderangle - for animating the border's gradient angle - styles: once (default), loop

Basically it spins around and you can configure the speed.

**Comment by YaLTeR** [comment]
Note that animating the niri gradient angle (which is the same as CSS linear-gradient angle) *does not* produce a smooth rotating animation:

https://github.com/user-attachments/assets/f3794073-ff13-4f48-bf85-9bf65e464a49

**Comment by beckend** [comment]
Can we then get to decide the animation function so the speed is adjusted dynamically. Either way, looks fine/good.
